changelogs/unreleased/igrishnov/gh-7479-bug-fix-uri-lib.md

```diff
@@ -0,0 +1,6 @@
+## bugfix/uri
+
+* Supplement the implementation of the `src/lib/uri` parser.
+  Before this fix a call `uri.parse(uri.format(uri.parse(3301)))`
+  returned an error of 'Incorrect URI'.
+  Now this call return correct `service: '3301'` (gh-7479).
```

Comment:
First, changelogs should state changes in Past Simple ("Supplemet**ed** the implementation ...").

Second, we should emphasize what the issue was about. I'd rephrase to something like this:

- Fixed a bug in the URI parser, because of which tarantoolctl failed to connect when the host name was skipped (gh-7479).

Reply:
Done